.vscode/launch.json

```diff
@@ -0,0 +1,168 @@
+{
```

Comment:
I was torn on this- I figured it should be committed because it's useful if you happen to be using that editor (or if I switch computers) but I do see the opposite argument as well.